scargo/scargo_src/sc_update.py

```diff
@@ -79,14 +80,18 @@ def scargo_update(config_file_path: Path) -> None:
     generate_conanfile(config)
 
     if target.family == "esp32":
+        esp32_config = config.esp32
+        if not esp32_config:
+            logger.error("No [esp32] in scargo config!")
+            sys.exit(1)
```

Comment:
same

Reply:
same what?

Reply:
sorry, I didn't realize that github hid the comment you were refering to.

so: same, outside the scope

Reply:
Validation added